docs/wcs/index.rst

```diff
@@ -235,6 +235,29 @@ ability to use the :class:`~astropy.wcs.WCS` to define projections in
 Matplotlib. More information on installing and using WCSAxes can be found `here
 <http://wcsaxes.readthedocs.org>`__.
 
+.. plot::
+    :include-source:
+
+    import numpy as np
+    from matplotlib import rcParams, pyplot as plt
+    from astropy.io import fits
+    from astropy.wcs import WCS
+    from astropy.visualization import astropy_mpl_style
+    rcParams.update(astropy_mpl_style)
```

Comment:
Once  #4031 is merged, you don't have to set `astropy_mpl_style` explicitly anymore in the docs.